packages/sdk/src/evm/schema/contracts/common/index.ts

```diff
@@ -85,7 +85,7 @@ export const CommonTrustedForwarderSchema = z.object({
  * @internal
  */
 export const CommonSymbolSchema = z.object({
-  symbol: z.string().optional().default(""),
+  symbol: z.string().default(""),
```

Comment:
default = behaves like optional but defaults if empty, so no reason to chain with optional